content/blog/p3-some-assembly-required/index.md

```diff
@@ -0,0 +1,327 @@
+* **Consistency**: [component resources](https://www.pulumi.com/learn/abstraction-encapsulation/component-resources/), [organization templates](https://www.pulumi.com/docs/pulumi-cloud/developer-portals/templates/), [drift detection](https://www.pulumi.com/docs/pulumi-cloud/deployments/drift/)
+* **Reproducibility**: [stacks](https://www.pulumi.com/learn/building-with-pulumi/understanding-stacks/), [deployments](https://www.pulumi.com/docs/pulumi-cloud/deployments/), [versioned data](https://www.pulumi.com/ai/answers/xig35anR7ibjAP5MhHDyxC/time-travel-queries-on-snowflake-dynamic-tables)
+* **Visibility**: [Pulumi Insights](https://www.pulumi.com/product/pulumi-insights/), [Pulumi Copilot](https://www.pulumi.com/product/copilot/)
+* **Security and Compliance**: [RBAC](https://www.pulumi.com/docs/pulumi-cloud/access-management/teams/), [GitHub Teams](https://www.pulumi.com/docs/pulumi-cloud/access-management/teams/#github-based-teams), [SAML-SSO](https://www.pulumi.com/docs/pulumi-cloud/access-management/saml/), [Pulumi ESC](https://www.pulumi.com/product/esc/), [Pulumi Crossguard](https://www.pulumi.com/crossguard/) 
+* **Auditability**: [audit logging](https://www.pulumi.com/docs/pulumi-cloud/audit-logs/)
+* **Developer Experience**: [Python/Go/JavaScript/C#](https://www.pulumi.com/docs/languages-sdks/), [popular IDE support](https://www.pulumi.com/blog/next-level-iac-breakpoint-debugging/), [command-line tools](https://www.pulumi.com/docs/cli/), [deeply hackable](https://www.pulumi.com/automation/)
+
+That’s all great, and much of that is already built-into Pulumi without the need for you to do anything at all. But not all of it. What parts do you actually need to set up and configure? And exactly how do you do that? That’s what this and the following posts in the series will be all about. 
+
+So to answer the question of what parts do you need? Here’s the bill of materials (BOM) to set up your own instance of Pulumi P3:
+
+### Bill of Materials:
+* **Authentication and Identity Management**:
+    * A GitHub organization that matches your Pulumi Cloud organization
+    * GitHub Teams users and roles that match your organizational structure and security needs
+
+* **Secrets, Configuration, and Policy**:
+    * Pulumi ESC environments to manage secrets across clouds
+    * Pulumi Crossguard policy packs that capture your company policies
+
+* **Developer Experience**:
+    * A set of reusable multi-language components for cross-cutting concerns/common services
+    * A set of organization templates that match your common use cases
+
+Let’s go through each of those and briefly discuss what it looks like to set that up. 
+
+## Authentication and identity management
+
+We highly recommend using GitHub for code management. So much so that we have deeply integrated GitHub into Pulumi Cloud across a number of features. While we support alternatives, this will be the easiest and more feature-rich way to configure your platform.
```

Comment:
Perhaps say Github is an excellent choice for code management, as we're doing a ton w/Gitlab as well

Reply:
edited